gauges/vacuum.go

```diff
@@ -94,3 +96,25 @@ func (g *Gauges) LastTimeAutoVacuumRan() *prometheus.GaugeVec {
 
 	return gauge
 }
+
+// VacuumRunningTotal returns the number of backends (including autovacuum worker processes)
+// that is currently running a vacuuming (not include VACUUM FULL).
+//
+// This metric is only supported for PostgreSQL 9.6 or newer versions
+func (g *Gauges) VacuumRunningTotal() prometheus.Gauge {
+	var gaugeOpts = prometheus.GaugeOpts{
+		Name:        "postgresql_vacuum_running_total",
+		Help:        "Number of backends (including autovacuum worker processes) that is currently vacuuming",
```

Comment:
`Number of backends (including autovacuum worker processes) currently vacuuming` maybe?